recipe/meta.yaml

```
@@ -1,31 +1,28 @@
 {% set name = "jellyfish" %}
-{% set version = "0.9.0" %}
-{% set bundle = "tar.gz" %}
-{% set hash = "40c9a2ffd8bd3016f7611d424120442f627f56d518a106847dc93f0ead6ad79a" %}
-
+{% set version = "1.0.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.{{ bundle }}
-  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
-  sha256: {{ hash }}
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f664520189e3ae16e0b5fee21f19d15c8926e2320fd195ccc65d96390d9e9689
 
 build:
   number: 0
-  skip: true  # [py<36]
-  script: {{ PYTHON }} -m pip install . -vv
+  # maturin isn't available on s390x
+  skip: true  # [py<37 or s390x]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   build:
     - {{ compiler('c') }}
+    - {{ compiler('rust') }}
   host:
     - pip
     - python
-    - setuptools
-    - wheel
+    - maturin
```

```suggestion
    - maturin >=0.14,<2
```
imho keeping ranges still makes sense when there is an upper bound